samples/rust/src/main.rs

```diff
@@ -113,6 +113,16 @@ fn main() {
         glean_metrics::party::drinks.test_get_num_recorded_errors(ErrorType::InvalidValue)
     );
 
+    {
+        let mut buffer = glean_metrics::test_metrics::timings.start_buffer();
+
+        for _ in 0..100 {
+            let now = Instant::now();
+            thread::sleep(Duration::from_millis(10));
+            buffer.accumulate(now.elapsed().as_millis() as u64);
```

Comment:
Any reason for using `now` and `now.elapsed` and all this `sleep` instead of having a loop-invariant `Duration::from_millis(10)` that we then `as_millis() as u64`? We don't need to involve clocks at all, I think.

Reply:
I don't even know anymore why I was sleeping here. To make it "realistic"? No idea, fixed values work just as good.